docs/tutorial/object_decomp.md

```diff
@@ -1,14 +1,161 @@
-# Analysing object files
+# Object Decompilation
 
 Up: [Contents](contents.md)
+
 Previous: [Documenting](documenting.md)
 
-TODO: To be written once automated objects are present.
+Object decompilation is the process of taking an object file and writing instructions to extract the various assets from it in the correct format, with appropriate labelling to distinguish their nature and/or use.
+
+## What is an object file?
+
+An object file is generally where most of the information used by overlays is stored. Its contents can include
+
+- Vertices (positional/normal/color data used by displaylists)
+- Textures
+- DisplayLists (instructions to the graphics processor on how to put together textures and vertices)
+- Skeleton (The underlying structure of an actor's shape, that can be manipulated to change its "pose")
+- Animations
+- Texture Animations
+
+## How we work with objects
+
+Because these are regarded as copyrighted, we do not want them in the repository. We instead extract them from the ROM. To do this we use a system called ZAPD (Zelda Asset Processor for Decompilation). The main aim of object decompilation is to give ZAPD an XML file that tells it what the assets it is supposed to be extracting actually are, so it can put them in the right format and name them.
```

Comment:
I feel like this should be worded slightly differently, but can't quite put my finger on how.

Reply:
I don't know if this what you are thinking Elliptic, but I think it should follow the lead of the `README` and refer to these as assets instead of mentioning copyright. Maybe something like

"These objects are considered assets so they are not included in the repository directly. Instead we use the Zelda Asset Processor for Decompilation (ZAPD) to extract them from ROM. As input, ZAPD takes an XML file that describes what and how to extract them. Each object already has an autogenerated xml created in the `assets/xml/objects` directory, though they are incomplete with just autogenerated names. Thus the goal of object decompilation is to complete these files by identifying any remaining blobs and giving everything proper names."